README.md

```diff
@@ -106,7 +106,7 @@ import { SubscribeModule } from '@ngneat/subscribe';
 
 @Component({
   standalone: true,
-  imports: [CommonModule, SpinnerComponent, SubscribeModule],
+  imports: [NgIf, NgFor, SpinnerComponent, SubscribeModule],
```

Comment:
issue: `*ngFor=...` is exposed via `NgForOf` instead of just `NgFor`. There's plan to expose an alias `NgFor` for `*ngFor=...` but that's not available yet.

Reply:
Thanks bro, I forgot about this one 💯